bundles/com.amazonaws.eclipse.sdk.ui/samples/AmazonS3TransferProgress/S3TransferProgressSample.java

```diff
@@ -130,6 +130,7 @@ public void progressChanged(ProgressEvent progressEvent) {
 
                     pb.setValue((int)upload.getProgress().getPercentTransferred());
 
+                    progressEvent.get
```

Comment:
Does this have side effects or can it be deleted?

Reply:
uh how is this not breaking the build (I did a full build just now), fixed